### PR TITLE
send email when regional restrictions updated

### LINKED ISSUE #16022: email amo-admins when AddonRegionRestrictionsAdmin is used
follow-up for #15968 to add the email functionality

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #16022

### CODE REVIEW

File: src/olympia/addons/admin.py [hunk 1]
```diff
@@ -375,6 +376,23 @@ def addon__name(self, obj):
...
     addon__name.short_description = 'Addon'
 
+    def _send_mail(self, obj, action):
+        message = (
+            f'Regional restriction for addon "{obj.addon.name}" '
+            f'[{obj.addon.id}] {action}: {obj.excluded_regions}')
+        send_mail(
+            f'Regional Restriction {action} for Add-on',
+            message,
+            recipient_list=('amo-admins@mozilla.com',))
```
Comment (bobsilverberg) [inline src/olympia/addons/admin.py:386]: Is this email address available as a constant somewhere?

File: src/olympia/addons/tests/test_admin.py [hunk 2]
```diff
@@ -766,40 +767,55 @@ def test_can_add(self):
 
         response = self.client.post(
             self.add_url, {
-                'excluded_regions': '["de", "pt-BR"]',
+                'excluded_regions': '["DE", "BR"]',
                 'addon': addon.id},
             follow=True)
         assert response.status_code == 200
+        # print(response.content)
```
Comment (bobsilverberg) [inline src/olympia/addons/tests/test_admin.py:774]: Did you mean to leave this `print` statement here?

**Review by bobsilverberg — APPROVED** [review]
Looks good. 

r+wc

File: src/olympia/addons/admin.py (hunk 1 above)
Comment (eviljeff) [reply src/olympia/addons/admin.py:386]: you'd think so, but we appear to define it directly everywhere it's used (in 7 other places)

File: src/olympia/addons/tests/test_admin.py (hunk 2 above)
Comment (eviljeff) [reply src/olympia/addons/tests/test_admin.py:774]: 🙈

Comment (eviljeff) [reply src/olympia/addons/tests/test_admin.py]: https://github.com/mozilla/addons-server/pull/16027/commits/10c983f80efcb5d76e31acc854eb22cf442051fd

**Pull request merged**